apiary.apib

```diff
@@ -3383,6 +3384,34 @@ Update a draft invoice. Booked invoices cannot be updated.
 
 + Response 204
 
+### invoices.updateBooked [POST /invoices.update]
+
+Update a booked invoice. Only available if "Booked invoices can't be edited" is disabled in the settings.
```

Comment:
Do you think we should make reference to the setting instead of a hard-coded label that can change?